.github/workflows/phpunit.yml

```diff
@@ -17,7 +17,7 @@ jobs:
         uses: actions/checkout@master
       - name: Install Dependencies
         run: |
-          bash bin/install-wp-tests.sh wordpress_test root root localhost ${{ matrix.wordpress_versions }}
+          bash bin/install-wp-cli-tests.sh wordpress_test root root localhost ${{ matrix.wordpress_versions }}
```

Comment:
```suggestion
          bash bin/cli-install-wp-tests.sh wordpress_test root root localhost ${{ matrix.wordpress_versions }}
```

Reply:
Not fixed because we also install the cli

Reply:
Fixed